src/test/scala/uk/gov/nationalarchives/DynamoFormattersTest.scala

```diff
@@ -84,11 +126,27 @@ class DynamoFormattersTest extends AnyFlatSpec with TableDrivenPropertyChecks wi
       invalidTypeAttributeValue,
       "'type': could not be converted to desired type: java.lang.Exception: Type Invalid not found"
     ),
+    (
+      invalidListOfStringsValue(originalFiles),
+      "'originalFiles': could not be converted to desired type: java.lang.RuntimeException: Cannot parse " +
```

Comment:
If you make the changes above then these two test names change. This one is
```scala
      "'originalFiles': could not be converted to desired type: java.lang.RuntimeException: Cannot parse " +
        "aString for field originalFiles into class java.util.UUID, 'originalFiles': missing"
```
This is the slightly different to what you had before. It's showing as missing because it's a number field but I think considering you're checking for string fields and this one isn't, that makes sense. 

The originalMetadataFiles one is now:
```scala
      "'originalMetadataFiles': could not be converted to desired type: java.lang.RuntimeException: Cannot parse " +
        "notAUuid for field originalMetadataFiles into class java.util.UUID"
```

Reply:
In your original test you had "1" in an `n()` field instead of `s()` With the changes I made, it's only looking for `s()` so if the value is under `n()` then you get the `missing` error.